src/tests/tracing/eventpipe/common/IpcUnixDomain.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+
+namespace Tracing.Tests.Common
+{
+    internal sealed class IpcUnixDomainSocket : IpcSocket
+    {
+        // Do we really need this class and do we need Dispose??
```

Comment:
Do we really need this, it seems like _ownsSocketFile and _path aren't used for these tests.